contributing.md

```diff
@@ -0,0 +1,31 @@
+Unless you explicitly state otherwise, any contribution intentionally submitted
+for inclusion in the work by you, as defined in the Apache-2.0 license, shall be
+dual licensed as above, without any additional terms or conditions.
```

Comment:
I think this needs to say "as stated in the `README.md`"

Reply:
Done.